### PR TITLE
HBW-152 Improve scroll style in select_table

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

**Comment by mxsmirnoff** [comment]
Jul-08-2019 16-03-53

File: hbw/app/assets/stylesheets/hbw.sass [hunk 1]
```diff
@@ -17,9 +17,6 @@
 
   @import 'tooltip'
 
-  .pre-scrollable
```
Comment (TimAle) [inline hbw/app/assets/stylesheets/hbw.sass]: What will happen with people using this class in their form configs?

File: hbw/app/assets/stylesheets/hbw.sass [hunk 2]
```diff
@@ -244,6 +241,34 @@
...
+    max-height: 700px
+
+    table
+      border-top: 1px
+
+    thead th
+      position: sticky
+      top: 0
+      background: white
+      box-shadow: inset 0 1px 0 #ecf0f1, inset 0 -1px 0 #ecf0f1
```
Comment (TimAle) [inline hbw/app/assets/stylesheets/hbw.sass]: Wow, do we really need this sophisticated shadow here?

**Comment by mxsmirnoff** [comment]
Updated

**Pull request merged**